docs/advanced/vgpusupport.md

```diff
@@ -0,0 +1,212 @@
+![](/img/v1.3/advanced/sriovgpudevices-enabled.png)
+
+3. Users can now navigate to the `vGPU Devices` page and view the associated `vgpudevices.devices.harvesterhci.io` objects.
+:::note
+It can take upto a few minutes for the vGPU Devices to be scanned by the pcidevices controller and be available in the UI
+:::
+![](/img/v1.3/advanced/vgpudevicelist.png)
+
+4. Users can now select a vGPU and configure an available profile from list of support profiles from the drop-down menu
+![](/img/v1.3/advanced/vgpuprofiles.png)
+:::note
+The list of available profiles is dependent on GPU and is read from the underlying /sys tree of the host. Please refer to the [NVIDIA documentation](https://docs.nvidia.com/grid/15.0/grid-vgpu-user-guide/index.html#supported-gpus-grid-vgpu) to view the various profiles and associated capabilities available.
+Once the first vGPU profile is selected, the NVIDIA driver automatically reconfigures the remaining vGPU's with the list of vGPU profiles available.
+:::
+
+5. Once a vGPU device has been setup, it can be attached to a new or existing VM
+![](/img/v1.3/advanced/vgpuattachment.png)
+
+Once the VM boots up, the vGPU will be available in the guest for usage by the workload. Additional vGPU drivers may be needed for your guest.
+
+:::note
+Once a vGPU has been assigned to a VM, it may not be possible to disable the same until the vGPU is removed from the VM.
+:::
+
+
+### Limitations
+
+#### Attaching multiple vGPU's:
+
+Known issue: https://github.com/harvester/harvester/issues/5289
+Attaching multiple vGPU's to a VM may fail for 2 possible reasons:
+
+1. Not all vGPU profiles support attachment of multiple vGPU's. Please refer to the [NVIDIA documentation](https://docs.nvidia.com/grid/16.0/grid-vgpu-release-notes-generic-linux-kvm/index.html) to identify the correct vGPU profile that supports the same. For example, we use NVIDIA A2/A16 GPU's and only Q-series vGPU's support attaching multiple vGPUs
+![](/img/v1.3/advanced/multiplevgpu.png)
+
+2. Once the correct profile is selected, only 1 GPU device in the VM definition can have `ramFB` enabled, and user needs to edit their VM as yaml and add the `virtualGPUOptions` to all non primary vGPU devices:
+```
+virtualGPUOptions:
+  display:
+     ramFB:
+       enabled: false 
+```
+Post this change the guest should be able to use the multiple vGPU's 
+
+
+#### Unable to use all vGPUs
+By default when vGPU support is enabled on a GPU, the NVIDIA driver creates 16 vGPU devices. Based on the first profile selected, the NVIDIA driver automatically configures the profiles available for remaining vGPUs.
+
+The profile used also dictates the maximum number of vGPUs available per GPU. Once the max count is exhausted then the remaining vGPU devices will have no profile available in their drop down menu and cannot be configured.
+
+For example, on our NVIDIA A2 GPU, using the `NVIDIA A2-4Q` profile means we can only have 4 vGPUs setup.
```

Comment:
Maybe we could put the link here https://docs.nvidia.com/grid/15.0/grid-vgpu-user-guide/index.html#vgpu-types-nvidia-a2